documentation/book/proc-configuring-kafka-mirror-maker-authentication-tls.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='proc-configuring-kafka-mirror-maker-authentication-tls-{context}']
+= Configuring TLS client authentication in Kafka Mirror Maker
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator with a `tls` listener with `tls` authentication enabled
+
+.Procedure
+
+Because Kafka Mirror Maker connects to two Kafka clusters (source and target), it is possible to configure TLS client authentication for one of them or both.
```

Comment:
Rewrite as suggested in the "Configuring SCRAM-SHA-512 authentication in Kafka Mirror Maker" module.